README.md

```diff
@@ -45,13 +54,246 @@ Each part of Data Driven Testing is designed to handle a specific ICU version.
+  e.g., ICU76 or CLDR76. A platform usually is implemented in multiple ICU
+  versions. Platforms should be updated to inclue3d each new ICU release.
+
 
 ## Architectural Overview
 
-Conceptually, there are three main functional units of the DDT implementation:
+Conceptually, there are four main functional units of the DDT implementation:
 
 ![Conceptual model of Data Driven Testing](./ddt_concept_model.png)
 
+# How to use Conformance Testing on your computer
+
+## Installing Comformance Testing
+
+The Conformance projectis publicly available. It can be installed and executed
+on most computer systems. Using GitHub, developers can investigate how the
+executors run each of the components. And developers can make changes and create
+pull requests to suggest updates to the projectd.
+
+### Get the source from GitHub
+
+1. Create a directory on the computer for installing and running
+   Conformance. Note that this may require more that 4 Gbytes when all the tests
+   are executed.
+
+1. Next, get the code and data from the [Unicode Conformance GitHub
+   site](https://github.com/unicode-org/conformance).
+
+1. From the green Code button on Git but, select one of the options for cloning the
+project, or download the project as a ZIP file.
+
+1. Unzip if needed. Then make sure that the directory contains directories for testdriver, testgen, schema, executors, verifier.
+
+
+### Set up tools and execution environments
+Some setup is requirel to run Data Driven Testing code locally:
+
+- Install the Python package `jsonschema`
+    * In a standard Python environment, you can run
+        ```
+        pip install jsonschema
+        ```
+    * Some operating systems (ex: Debian) might prefer that you install
+    the OS package that encapsulates the Python package
+        ```
+        sudo apt-get install python-jsonschema
+        ```
+
+Note: Setting up Python modules or other software may be better done using a
+virtual environment such as using `pip env` for installation.
+
+- Install the minimum version supported by ICU4X
+    * The latest minimum supported supported Rust version ("MSRV") can be found in the
+    [`rust-toolchain.toml` file](https://github.com/unicode-org/icu4x/blob/main/rust-toolchain.toml)
+    * To view your current default Rust version (and other locally installed Rust versions):
+        ```
+        rustup show
+        ```
+    * To update to the latest Rust version:
+        ```
+        rustup update
+        ```
+  - Install `logrotate`
+      ```
+      sudo apt-get install logrotate
+      ```
+
+### Running End-To-End on a local computer
+
+Running conformance is easy. Simply execute the script that first generates
+data, runs executors, checks data against schema, and then runs the verifier,
+creating the dashboard.
+
+The standard script runs all components on all platforms with all ICU
+version. Note that this takes a few minutes.
+
+`` bash generateDataAndRun.sh ``
```

Comment:
Make this a code fence by doing a triple backquote on a separate line before and after the command